hackage-server.cabal

```diff
@@ -135,7 +135,7 @@ common defaults
     , scientific
   -- other dependencies shared by most components
   build-depends:
-    , aeson                 ^>= 2.2.0.0
+    , aeson                  >= 2.1.0.0
```

Comment:
All dependencies have upper bounds here, so if you switch from `^>=` to `>=` you need an upper bound, like `>= 2.1.0.0 && < 2.3`.
Alternatively, you can list just both tested versions: `^>= 2.1.0.0 || ^>= 2.2.0.0`